tempesta_fw/t/unit/test_hash.c

```diff
@@ -166,7 +166,7 @@ TEST(tfw_hash_str, distributes_all_input_across_hash_bits)
 {
 	char buf[31]; /* maximum tail */
 	TfwStr str = {
-		.ptr = buf,
+		.data = buf,
```

Comment:
There are macros `DEFINE_TFW_STR()` and `TFW_STR_FROM()` that can be used for this purpose.